packages/reactiveData/reactiveData.0.2/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "1.2"
+
+maintainer: "Hugo Heuzard <hugo.heuzard@gmail.com>"
+authors: ["Hugo Heuzard <hugo.heuzard@gmail.com>"]
+homepage: "https://github.com/ocsigen/reactiveData"
+dev-repo: "https://github.com/ocsigen/reactiveData.git"
+bug-reports: "https://github.com/ocsigen/reactiveData/issues"
+
+doc:"http://ocsigen.github.io/reactiveData/0.2/"
+
+tags: [ "reactive" "declarative" "signal" "event" "frp" ]
+license: "LGPL-3.0 with OCaml linking exception"
+
+depends: ["ocamlfind" "react"]
```

Comment:
`ocamlfind` is probably a `{build}` dependency and `ocamlbuild` is missing but I'll merge this anyway.